README.md

```diff
@@ -107,14 +178,24 @@ Imagine that the squares of an ordinary checkerboard are numbered in two differe
 
 **a)**  Given a particular R and C, find the corresponding N.  That is, write an expression for variable N containing variables R and C.
 
+var r = 7;
+
+var c = 7;
+
+r * c = n + 7 * 2
+
```

Comment:
You wouldn't want to define the variables by using numbers here. Instead, we can assume that R represents the rows, that c represents the columns, and that n is a number from 0-63, which represents a number in each individual checker  square. It really helps to envision this if you draw a picture so that you can get a solid mental image of what's going on - something I would recommend doing whenever possible when it comes to coding. 

Here is Dan's solution to this problem: n = 8*r + c; So, if we multiply 8 by the row, and then add the column to it, we'll get the correct number placement on the board. So, for example, if we use (1,1) for (R,C), the solution would appear as follows: 8*1 + 1 = 9 - which would be correct, because the (R,C) of (0,1) is in number 9 on the checkerboard. 

This might be confusing, and if so, I'd be more than happy to explain in class or in office hours!!